src/lib/Cropper.svelte

```diff
@@ -17,6 +17,8 @@
   export let crossOrigin: HTMLImgAttributes['crossorigin'] = null
   export let restrictPosition = true
   export let tabindex: number | undefined = undefined
+  export let objectFit: ObjectFit = 'contain'
+  export let mediaObjectFit: ObjectFit = objectFit
```

Comment:
I don't think `mediaObjectFit` should be exported as it's an property that is computed internally.